src/codec2_fft.h

```diff
@@ -14,10 +14,11 @@
 #include <string.h>
 #include <math.h>
 
-#ifdef FDV_ARM_MATH
-  #include "fdv_arm_math.h"
+#ifndef FDV_ARM_MATH
+  #define USE_KISS_FFT
 #else
-    #define USE_KISS_FFT
+  #include "arm_math.h"
+  #include "arm_const_structs.h"
 #endif
```

Comment:
@tmiw can you pls tell me what these includes are for?

Reply:
These are the includes that were needed to have the CMSIS functions used by FDV_ARM_MATH work. I'm not sure if fdv_arm_math.h had anything else previously.

Reply:
Thanks.  So is `FDV_ARM_MATH` disabled for the stm32 build?  I think that's the case, I'm poking around the stm32 folders and can't see it enabled :thinking:

Reply:
Correct, it's not used for the SM1000.

Reply:
I ran the stm32 ctests, once I merged with master they passed OK :+1